mkosi

```diff
@@ -1767,7 +1892,7 @@ def build_image(args, workspace, run_build_script):
     # If there's no build script set, there's no point in executing
     # the build script iteration. Let's quite early.
     if args.build_script is None and run_build_script:
-        return (None, None)
+        return (None, None, None)
```

Comment:
This really calls for a `namedtuple`, but let's do that in a cleanup PR once this one lands.